prime-router/build.gradle.kts

```diff
@@ -829,7 +829,7 @@ dependencies {
     implementation("org.yaml:snakeyaml:2.2")
     implementation("io.github.linuxforhealth:hl7v2-fhir-converter") {
         version {
-            branch = "master"
+            branch = "jalbinson/add-prt-as-supported-segment" // TODO: switch me back to master when PR is merged
```

Comment:
PR for our forked library
https://github.com/focusconsulting/hl7v2-fhir-converter/pull/7

After that gets merged this can be changed back to master again.